sdk/eventhub/event-hubs/test/public/eventHubConsumerClient.spec.ts

```diff
@@ -810,7 +810,7 @@ describe("EventHubConsumerClient", () => {
       let subscription: Subscription | undefined;
       await new Promise<void>((resolve, reject) => {
         subscription = consumerClient.subscribe(
-          // @ts-expect-error
+          // @ts-expect-error number for partitionId should work even if type is string
```

Comment:
I thought I was going crazy because I _know_ I fixed these:
https://github.com/Azure/azure-sdk-for-js/pull/13285/files#diff-55397d6ab644a9c4b90887b375e3a6095bea4e388a28f8a3fe3649082daf5539R66

Must have gotten lost in some merge conflicts.

Reply:
hehe